source/openapi-admin-v3.yaml

```diff
@@ -3064,6 +3064,201 @@ paths:
+                items:
+                  "$ref": "#/components/schemas/CustomResolver"
+    post:
+      tags:
+        - graphql
+      operationId: adminCreateCustomResolver
+      summary: Create a custom resolver
+      description: Create a new [custom resolver](https://www.mongodb.com/docs/realm/graphql/custom-resolvers/).
+      requestBody:
+        description: A valid [custom resolver configuration](https://www.mongodb.com/docs/realm/manage-apps/configure/config/graphql/#appconfig-custom-resolver) object.
+        required: true
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/CustomResolver"
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/CustomResolver"
+  /groups/{groupId}/apps/{appId}/graphql/custom_resolvers/{customResolverId}:
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/CustomResolverId"
+    get:
+      tags:
+        - graphql
+      operationId: adminGetCustomResolver
+      summary: Get a custom resolver
+      description: Get a specific [custom resolver](https://www.mongodb.com/docs/realm/graphql/custom-resolvers/) configuration.
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/CustomResolver"
+    put:
+      tags:
+        - graphql
+      operationId: adminModifyCustomResolver
+      summary: Modify a custom resolver
+      description: Modify an existing [custom resolver](https://www.mongodb.com/docs/realm/graphql/custom-resolvers/) configuration.
+      requestBody:
+        description: A valid, updated [custom resolver configuration](https://www.mongodb.com/docs/realm/manage-apps/configure/config/graphql/#appconfig-custom-resolver) object.
+        required: true
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/CustomResolver"
+      responses:
+        "204":
+          description: Updated
+    delete:
+      tags:
+        - graphql
+      operationId: adminDeleteCustomResolver
+      summary: Delete a custom resolver
+      description: Delete an existing [custom resolver](https://www.mongodb.com/docs/realm/graphql/custom-resolvers/) configuration.
+      responses:
+        "204":
+          description: Deleted
+
+  /groups/{groupId}/apps/{appId}/validation_settings/graphql:
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+    get:
+      tags:
+        - graphql
+      operationId: adminGetValidationSettings
+      summary: Get validation settings
+      description: Get the current validation level and action for reads and writes.
```

Comment:
what are 'validation settings'? can you link to relevant docs here?

Reply:
Nope we don't have docs for it because this lowkey didn't work when we released it (and I'm not sure if we ever fixed it)